Tests/Test_Amperes_Task.c

```diff
@@ -59,67 +59,46 @@ void Task1(void *p_arg){
                 &err);
     assertOSError(err);
 
-    RTOS_BPS_MutexCreate(&WDog_Mutex, "Watchdog Mutex");
+    OSMutexCreate(&WDog_Mutex,
```

Comment:
Is there a reason this was changed to the older version?

Reply:
Oh I accidentlly messed up this file so I copy pasted it from main, my bad.